### ISSUE TITLE
DDIM inversion or ramdon noise?

### ISSUE THREAD

**Issue by tszFung-gz** [showlab/Tune-A-Video#73]
Excellent work! 
I have a question to ask, is this DDIM inversion necessary in the process of diffusion of video data? I tried to use DPM inversion and got seriously wrong results. I tried to use DDIM inversion and got seriously wrong style, though the better temporal consistency. What other samplers are available?

**Comment by zhangjiewu** [comment]
Yes, DDIM inversion necessary to preserve the motion in source video. DPM would fail as it is not deterministic. For style change, reducing finetuning steps may help.

**Comment by tszFung-gz** [comment]
Thanks for the answer~
